src/Sql/Sql/Instance Pools/Cmdlet/GetAzureSqlInstancePool.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.Instance_Pools.Model;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+
+namespace Microsoft.Azure.Commands.Sql.Instance_Pools.Cmdlet
+{
+    /// <summary>
+    /// Defines the Get-AzRmSqlInstancePool cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstancePool")]
```

Comment:
@johnpaulkee since this cmdlet has multiple parameter sets, please make sure that a `DefaultParameterSetName` is defined in this cmdlet attribute (it should be `ListBySubOrResourceGroupParameterSet`)